### PR TITLE
support for Aurora DAO tokens

### PR DESCRIPTION
**Pull request by hershman**
Added support for the Aurora DAO tokens AURA and IDXM

### CODE REVIEW

**Comment by stesal** [comment]
+1

**Comment by zlajoni** [comment]
+1

File: config/main.json [hunk 1]
```diff
@@ -732,7 +732,9 @@
     { "addr": "0x85e076361cc813a908ff672f9bad1541474402b2", "name": "TEL", "decimals": 2 },
     { "addr": "0x4cc19356f2d37338b9802aa8e8fc58b0373296e7", "name": "SELFKEY", "decimals": 18 },
     { "addr": "0x9b7593aae6b48d02668808c2dfbfc4bef35ef957", "name": "DVN", "decimals": 18 },
-    { "addr": "0xa5F8fC0921880Cb7342368BD128eb8050442B1a1", "name": "ARY", "decimals": 18 }
+    { "addr": "0xa5F8fC0921880Cb7342368BD128eb8050442B1a1", "name": "ARY", "decimals": 18 },
+    { "addr": "0xcc13fc627effd6e35d2d2706ea3c4d7396c610ea", "name": "IDXM", "decimals": 8 },
```
Comment (freeatnet) [inline config/main.json]: Looks good ✔️

File: config/main.json [hunk 2]
```diff
@@ -732,7 +732,9 @@
...
+    { "addr": "0xcdcfc0f66c522fd086a1b725ea3c0eeb9f9e8814", "name": "AURA", "decimals": 18 }
```
Comment (freeatnet) [inline config/main.json]: Also looks good!

File: tokenGuides/AURA.ejs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<blockquote> 
+  <p>AURA is the Aurora utility token for the Aurora DAO.</p>
```
Comment (freeatnet) [inline tokenGuides/AURA.ejs]: I'd add "Users stake AURA to secure the network, and are rewarded with fees from IDEX and DC" here.

File: tokenGuides/IDXM.ejs [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<blockquote> 
+  <p>IDXM is the membership token for <a href="https://idex.market">https://idex.market</a> and can be used for free trades (until 2020) or 2x market making rewards.</p>
```
Comment (freeatnet) [inline tokenGuides/IDXM.ejs]: Let's drop `https://` from link text, ie:
```
IDXM is the membership token for <a href="https://idex.market">idex.market</a>
```

**Comment by hershman** [comment]
"allow edits from maintainers" seems to be check.  Do you want to make these changes or should I?

**Comment by freeatnet** [comment]
@hershman I'll rebase and merge the thing once you're done edits. JSON config is a bit too sensitive to changes.

**Comment by hershman** [comment]
@freeatnet Great - I made the text edits you requested.  Please let me know if you need anything else.

**Pull request merged**